cmd/app.go

```diff
@@ -352,6 +348,22 @@ func appUpgrade(ctx *cli.Context) error {
 	return c.ProjectClient.App.ActionUpgrade(app, au)
 }
 
+func getExternalIDInVersion(externalID string, version string) (string, error) {
+	u, err := url.Parse(externalID)
+	if err != nil {
+		return "", err
+	}
+
+	q := u.Query()
+	q.Set("version", version)
+	// Don't need encoding here because it as a whole is the value of the externalId filter
```

Comment:
This comment is confusing. It says that don't encode here but we got `q.Encode()` in the next line.

Reply:
Updated